content/a/ptw-awards.md

```diff
@@ -4,151 +4,151 @@ weight = 6
+The awards were originally planned to be hosted annually, at the end of the Year.
+However, so far only one edition has been held, in Dec 2022/Jan 2023, to summarize the first chapter of PTW's existence, from [PTW#1: Revolucja](@/e/ptw/2021-10-09-ptw-1-revolucja.md) until the end of 2022.
 Fans nominated their candidates by voting with reactions on PTW's Facebook page, which posted selected nominees in multiple categories.
 
 This award ceremony was held during [PTW Underground 12](@/e/ptw/2023-02-26-ptw-underground-12.md).
-Each of the winners was awarded a physical trophy, crafted out of acrylic plastic, depicting a wrestler figure flexing his muscles, joined with the "Kinguin PTW" logo, and individual signatures on the base.
+Each of the winners was awarded a physical trophy, crafted out of acrylic plastic, depicting a wrestler figure flexing his muscles against the "Kinguin PTW" logo, with individual signatures on the base.
 
-Although most of the fan-voting went uninterrupted, there was one category which brewed controversy.
+Although most of the fan-voting went uninterrupted, there was one category which caused controversy.
 While [Sędzia Seweryn](@/w/sedzia-seweryn.md) and Sędzia Matek fought a close battle for the prize, Sędzia Rafał unexpectedly and rapidly gained over 250 reactions.
 Until that moment he had just around 40, compared to his opponents' 150 reactions.
 After a short investigation, it was found that somebody decided to buy around 200 reactions from well-known Vietnamese social media bot farms.
-This escalated into the so-called "Afera Azjatycka" (_Asian Affair_), concluding in Sędzia Rafał's disqualification from the voting, and suspension from referee duties.
+This escalated into the so-called "Afera Azjatycka" (_Asian Scandal_), concluding in Sędzia Rafał's disqualification from the voting, and suspension from referee duties.
 His reputation among PTW's fans and other workers also suffered dramatically.
 
-Later it turned out that the entire thing was orchestrated as a prank by Szymon "Modzel" Modzelewski - a wrestling fan, who wanted to tease PTW and Pawłowski by interfering with the poll results.
+Later it turned out that the entire thing was orchestrated as a prank by Szymon "Modzel" Modzelewski - a wrestling fan who wanted to tease PTW and Pawłowski by interfering with the poll results.
 
 ## Categories awarded
 
 - Wrestler Roku (_Wrestler of the Year_)
 - Bohater Roku (_Hero of the Year_)
 - Złoczyńca Roku (_Villain of the Year_)
 - Debiutant Roku (_Newcomer of the Year_)
-- Mówca Roku (_Wrestler of the Year_)
-- Tag Team Roku (_Wrestler of the Year_)
-- Komentator Roku (_Wrestler of the Year_)
+- Mówca Roku (_Speaker of the Year_)
+- Tag Team Roku (_Tag Team of the Year_)
+- Komentator Roku (_Commentator of the Year_)
```

Comment:
Co tu się porobiło że tak to weszło 🙈

Reply:
Skutki uboczne studenckiej metody Kopiego/Paste'a. :D